bytecomp/lambda.mli

```diff
@@ -238,7 +240,7 @@ type lambda =
   | Llet of let_kind * Ident.t * lambda * lambda
   | Lletrec of (Ident.t * lambda) list * lambda
   | Lprim of primitive * lambda list * Location.t
-  | Lswitch of lambda * lambda_switch
+  | Lswitch of lambda * lambda_switch * switch_names option
 (* switch on strings, clauses are sorted by string order,
```

Comment:
would it be easier to put `switch_names option` into lambda_switch

Reply:
That's what I've tried before, among other things. It becomes pretty invasive if one does that.

Reply:
Actually, I was thinking of something else. This change looks like it simplifies things.

Reply:
Done